### PR TITLE
fix(ui): use absolute URL to redirect from autocomplete list. Closes #3903

### LINKED ISSUE #3903: Workflow auto-completes to wrong URL
## Summary 

Using the autocomplete box on the workflow page sends you to the wrong. Has this ever worked? If not, lets remove it.

autocomplete-bug

## Diagnostics

master

---

**Message from the maintainers**:

Impacted by this bug? Give it a 👍. We prioritise the issues with the most 👍.

### PR DESCRIPTION
**Pull request by nirav24**
Closes #3903

Checklist:

* [ ] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this is a chore.
* [ ] The title of the PR is (a) [conventional](https://www.conventionalcommits.org/en/v1.0.0/), (b) states what changed, and (c) suffixes the related issues number. E.g. `"fix(controller): Updates such and such. Fixes #1234"`.  
* [x] I've signed the CLA.
* [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [ ] My builds are green. Try syncing with master if they are not. 
* [ ] My organization is added to [USERS.md](https://github.com/argoproj/argo/blob/master/USERS.md).

### CODE REVIEW

File: ui/src/app/workflows/components/workflows-list/workflows-list.tsx [hunk 1]
```diff
@@ -414,7 +414,7 @@ export class WorkflowsList extends BasePage<RouteComponentProps<any>, State> {
                                 </React.Fragment>
                             )}
                             onSelect={val => {
-                                ctx.navigation.goto(`./${val}`);
+                                ctx.navigation.goto(`/workflows/${val}`);
```
Comment (alexec) [inline ui/src/app/workflows/components/workflows-list/workflows-list.tsx:417]: thank you for submitting this PR. I think, to support BASH_HREF, you need to use `uiUrl("workflows/"+val`?

**Comment by alexec** [comment]
`TestPendingRetryWorkflow` is flaky.

File: ui/src/app/workflows/components/workflows-list/workflows-list.tsx (hunk 1 above)
Comment (nirav24) [reply ui/src/app/workflows/components/workflows-list/workflows-list.tsx]: thanks @alexec  for pointing that out. I fixed in https://github.com/argoproj/argo/pull/3906/commits/5f97e8e58859d16b56246abb6125a3e55d01585d

**Comment by alexec** [comment]
No one has complained about this, so not sure it is much used, and therefore wont` backport to v2.10.

**Pull request merged**